compiler_opt/rl/log_reader_test.py

```diff
@@ -246,6 +275,40 @@ def test_seq_example_conversion(self):
 """, tf.train.SequenceExample())
     self.assertProtoEquals(expected_ctx_0, seq_examples['context_nr_0'])
 
+  def test_errors(self):
+    logfile = self.create_tempfile()
+    for i in range(1, len(LogTestExampleBuilder.ErrorMarkers)):
+      create_example(
+          logfile, introduce_errors_pos=LogTestExampleBuilder.ErrorMarkers(i))
+      with self.assertRaises(Exception):
```

Comment:
Can we be more specific about the exception here?

Reply:
Some are json, some are the log protocol. Listing them all would be tedious. We could wrap them, but how's that different from just saying "Exception"; finally they are individually not that interesting programmatically, insofar as I can tell, because regardless of the underlying error, programmatically there's little we can do. So the exceptions here matter as something to ultimately report to the user, but that's about it.